src/RazorSdk/SourceGenerators/RazorSourceGenerator.cs

```diff
@@ -252,11 +254,11 @@ private static IReadOnlyList<TagHelperDescriptor> GetTagHelperDescriptorsFromRef
             return tagHelperDescriptors;
         }
 
-        private static void PopulateAssemblyInfo(GeneratorExecutionContext context)
+        private static SourceText GetProvideApplicationPartFactorySourceText()
         {
             var typeInfo = "typeof(global::Microsoft.AspNetCore.Mvc.ApplicationParts.ConsolidatedAssemblyApplicationPartFactory)";
```

Comment:
```suggestion
            var typeInfo = "Microsoft.AspNetCore.Mvc.ApplicationParts.ConsolidatedAssemblyApplicationPartFactory, Microsoft.AspNetCore.Mvc.Razor";
```

Reply:
Yeah, I tried this in ba01d88 but it didn't work. I missed the need for `Microsoft.AspNetCore.Mvc.Razor`.